chia/full_node/mempool_check_conditions.py

```diff
@@ -416,7 +416,7 @@ def get_puzzle_and_solution_for_coin(generator: BlockGenerator, coin_name: bytes
     try:
         block_program = generator.program
         if not generator.generator_args:
-            block_program_args = [NIL]
+            block_program_args = [NIL_PROGRAM]
```

Comment:
we're assigning `block_program_args`, intuitively it seems correct to think of this as a `NIL` value, rather than a program (but I realize it's all the same). My point is, instead of renaming this constant, would it make more sense to set it to `SExp.from_bytes(b"\x80")`  below? It's exactly the same value.

Or does this need to have the type `Program` to pass mypy checks?